common/constants.h

```diff
@@ -0,0 +1,5 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+constexpr double kMegaByteSizeInBytes = (1 << 20);
```

Comment:
Please put this in the `flutter` namespace.

Reply:
Done